Integrations/integration-Carbon_Black_Enterprise_Live_Response.yml

```diff
@@ -1632,7 +1629,7 @@ script:
                 return directoryList();
             case 'cb-process-execute':
                 return processExec();
-            case 'cb-memdeump':
+            case 'cb-memdump':
```

Comment:
```suggestion
            case 'cb-memdeump':
```

Reply:
revert this back to the typo spelling in order to maintain backward compatibility.